specification/developerhub/resource-manager/Microsoft.DevHub/preview/2022-10-11-preview/workflow.json

```diff
@@ -0,0 +1,1008 @@
+          "title": "The Kubernetes namespace the application is deployed to"
+        },
+        "acr": {
+          "$ref": "#/definitions/ACR"
+        },
+        "oidcCredentials": {
+          "type": "object",
+          "description": "The fields needed for OIDC with GitHub.",
+          "properties": {
+            "azureClientId": {
+              "type": "string",
+              "description": "Azure Application Client ID"
+            },
+            "azureTenantId": {
+              "type": "string",
+              "description": "Azure Directory (tenant) ID"
+            }
+          }
+        },
+        "aksResourceId": {
+          "type": "string",
+          "description": "The Azure Kubernetes Cluster Resource the application will be deployed to.",
+          "title": "The Azure Kubernetes Managed Cluster resource."
+        },
+        "prURL": {
+          "type": "string",
+          "description": "The URL to the Pull Request submitted against the users repository.",
+          "readOnly": true
+        },
+        "pullNumber": {
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true,
+          "description": "The number associated with the submitted pull request."
+        },
+        "prStatus": {
+          "$ref": "#/definitions/PullRequestStatus"
+        },
+        "lastWorkflowRun": {
+          "$ref": "#/definitions/WorkflowRun"
+        },
+        "authStatus": {
+          "$ref": "#/definitions/AuthorizationStatus"
+        },
+        "artifactGenerationProperties": {
+          "$ref": "#/definitions/ArtifactGenerationProperties"
+        }
+      }
+    },
+    "WorkflowListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Workflow"
+          },
+          "description": "The list of workflows."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to the next set of workflow results.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from List Workflows operation."
+    },
+    "PullRequestStatus": {
+      "type": "string",
+      "readOnly": true,
+      "enum": [
+        "unknown",
+        "submitted",
+        "merged",
+        "removed"
+      ],
+      "x-ms-enum": {
+        "name": "PullRequestStatus",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "unknown",
+            "description": "Pull Request state unknown."
+          },
+          {
+            "value": "submitted",
+            "description": "Pull Request submitted to repository."
+          },
+          {
+            "value": "merged",
+            "description": "Pull Request merged into repository."
+          },
+          {
+            "value": "merged",
+            "description": "Workflow no longer found within repository."
```

Comment:
Is this intentional .. it looks like the value "merged" has been added twice ? and "removed" has been removed ?

Reply:
this is not intentional. Thank you for catching it!